### DISCUSSION TITLE
Add --host feature from Vite

### DISCUSSION THREAD

**Discussion by MauritsWilke** [vitest-dev/vitest#2237] [Ideas]
Hey all! I was wondering if vitest could get the same host feature vite has:
https://vitejs.dev/config/server-options.html

I want to use this in combination with the `--ui` flag because I usually run tests on my laptop but program on my PC.

**Comment by sheremet-va** [comment]
you can use --api: https://vitest.dev/config/#api

**Reply by MauritsWilke** [reply]
I have tried that but it doesn't allow me to host it and access it on laptop. Maybe i'm doing something wrong.
